clang/test/CIR/Transforms/lifetime-check-agg.cpp

```diff
@@ -1,4 +1,5 @@
 // RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -mconstructor-aliases -fclangir -clangir-disable-emit-cxx-default -fclangir-lifetime-check="history=all;remarks=all" -clangir-verify-diagnostics -emit-cir %s -o %t.cir
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -mconstructor-aliases -fclangir-analysis-only -fclangir-lifetime-check="history=all;remarks=all" %s -clangir-verify-diagnostics -emit-obj -o /dev/null
```

Comment:
Nice! I think we only need this extra RUN line for one of the tests, we likely don't need the compile time overhead for all tests.

Reply:
Done

Reply:
Sorry for the confusion, this is not what I asked. I rather we keep it as before your PR, my suggestion is:
- You add an extra `RUN` line for `clang/test/CIR/Transforms/lifetime-check-agg.cpp`
- Not touch the other existing tests at all. One place is enough to test the specifics of what you are looking for.

Reply:
Done. Sorry for misreading. It was clear.